sechub-web-ui/README.md

```diff
@@ -76,13 +78,27 @@ You can use the sechub-api.sh script to manage your user and project.
 
 Now you can test your web-ui with sechub and real scans!
 
-### Building for Production
+## Building for Production
+
+### Environment Variables
 
-Set Environment Variables:
 Be aware that `npm run build` sets the environment variables at build time.
 For deploying runtime ENV please se sechub-web-ui-solution/docker/nginx/conf.json it will override the VITE variables. Be aware that the config.json will be served by nginx.
```

Comment:
```suggestion
For deploying runtime ENV please se sechub-web-ui-solution/docker/nginx/config.json it will override the VITE variables. Be aware that the config.json will be served by nginx.
```